<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>20</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>61</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblWelcome">
          <property name="text">
           <string>Hosgeldiniz Mesajınızı Yazınız:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMessage">
          <property name="text">
           <string>Chat Sunucusuna hoşgeldiniz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>48</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port Numaranızı Seçiniz:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbPort">
          <property name="minimum">
           <number>2000</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>2999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="lblStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bağlantı kesildi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Başlat</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Durudur</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
